chat-commands.js

```diff
@@ -4154,13 +4156,15 @@ const commands = {
 		if (target === 'help' || target === 'h' || target === '?' || target === 'commands') {
 			this.sendReply("/help OR /h OR /? - Gives you help.");
 		} else if (!target) {
-			this.sendReply("COMMANDS: /msg, /reply, /logout, /challenge, /search, /rating, /whois");
-			this.sendReply("OPTION COMMANDS: /nick, /avatar, /ignore, /away, /back, /timestamps, /highlight");
-			this.sendReply("INFORMATIONAL COMMANDS: /data, /dexsearch, /movesearch, /itemsearch, /groups, /faq, /rules, /intro, /formatshelp, /othermetas, /learn, /analysis, /calc (replace / with ! to broadcast. Broadcasting requires: + % @ * # & ~)");
+			this.sendReply("COMMANDS: /msg, /reply, /logout, /challenge, /search, /rating, /whois, /user, /report, /join, /leave, /makegroupchat, /userauth, /roomauth");
+			this.sendReply("BATTLE ROOM COMMANDS: /savereplay, /hideroom, /ioo, /ionext, /invite, /timer, /forfeit");
```

Comment:
I would call this `/inviteonly` instead of `/ioo`. `/ioo` can be documented inside `/inviteonly`.

Reply:
Also `/ionext` should be `/inviteonlynext`; or maybe entirely left off and just documented inside `/inviteonly` as well.

Reply:
Also, with `/roomsettings` in existence, this might not strictly be necessary.

Reply:
`/inviteonlynext` does not exist as an alias of ionext (yet). Should that be changed?
 `/roomsettings` does not work inside battle rooms (and if it did, I'm pretty sure no one would use it to set modjoin because it's a lot clunkier than /ioo).